spec.emu

```diff
@@ -102,97 +112,170 @@ contributors: Google, Ecma International
             </tr>
           </thead>
           <tr>
-            <td>`"sunday"`</td>
-            <td>Name indicating Sunday, e.g. Su, Sun, Sunday, etc.</td>
-          </tr>
-          <tr>
-            <td>`"monday"`</td>
+            <td>`"1"`</td>
             <td>Name indicating Monday, e.g. M, Mon, Monday, etc.</td>
           </tr>
           <tr>
-            <td>`"tuesday"`</td>
+            <td>`"2"`</td>
             <td>Name indicating Tuesday, e.g. Tu, Tue, Tuesday, etc.</td>
           </tr>
           <tr>
-            <td>`"wednesday"`</td>
+            <td>`"3"`</td>
             <td>Name indicating Wednesday, e.g. W, Wed, Wednesday, etc.</td>
           </tr>
           <tr>
-            <td>`"thursday"`</td>
+            <td>`"4"`</td>
             <td>Name indicating Thursday, e.g. Th, Thur, Thursday, etc.</td>
           </tr>
           <tr>
-            <td>`"friday"`</td>
+            <td>`"5"`</td>
             <td>Name indicating Friday, e.g. F, Fri, Friday, etc.</td>
           </tr>
           <tr>
-            <td>`"saturday"`</td>
+            <td>`"6"`</td>
             <td>Name indicating Saturday, e.g. Sa, Sat, Saturday, etc.</td>
           </tr>
           <tr>
-            <td>`"january"`</td>
+            <td>`"7"`</td>
+            <td>Name indicating Sunday, e.g. Su, Sun, Sunday, etc.</td>
+          </tr>
+        </table>
+      </emu-table>
+
+    </emu-clause>
+
+    <emu-clause id="sec-isvalidmonthcode" aoid="IsValidMonthCode">
+      <h1>IsValidMonthCode ( _code_ )</h1>
+      <emu-alg>
+        1. If _code_ is listed in <emu-xref href="#table-validcodeformonth"></emu-xref>, then
+          1. Return *true*.
+        1. Else,
+          1. Return *false*.
+      </emu-alg>
+
+      <emu-table id="table-validcodeformonth">
+        <emu-caption>Codes For Month of DisplayNames</emu-caption>
+        <table class="real-table">
+          <thead>
+            <tr>
+              <th>Code</th>
+              <th>Description</th>
+            </tr>
+          </thead>
+          <tr>
+            <td>`"1"`</td>
             <td>Name indicating January, e.g. Jan, January, etc.</td>
```

Comment:
Why don't you say,

> Name indicating the first month of the year, e.g., Jan, January.

Then the definition is not tied to the Gregorian calendar only.

Reply:
DONE